flow360/new_simulation_models.py

```diff
@@ -0,0 +1,125 @@
+"""
+This module is flow360 for simulation based models
+"""
+
+from flow360.component.simulation import services
+from flow360.component.simulation import units as u
+from flow360.component.simulation.entity_info import GeometryEntityInfo
+from flow360.component.simulation.framework.param_utils import AssetCache
+from flow360.component.simulation.meshing_param.edge_params import (
+    HeightBasedRefinement,
+    SurfaceEdgeRefinement,
+)
+from flow360.component.simulation.meshing_param.face_params import SurfaceRefinement
+from flow360.component.simulation.meshing_param.params import (
+    MeshingDefaults,
+    MeshingParams,
+)
+from flow360.component.simulation.meshing_param.volume_params import AutomatedFarfield
+from flow360.component.simulation.models.solver_numerics import (
+    HeatEquationSolver,
+    LinearSolver,
+    NavierStokesSolver,
+    NoneSolver,
+    SpalartAllmaras,
+    TransitionModelSolverType,
+    TurbulenceModelSolverType,
+)
+from flow360.component.simulation.models.surface_models import (
+    Freestream,
+    Inflow,
+    Outflow,
+    Periodic,
+    SlipWall,
+    SymmetryPlane,
+    Wall,
+)
+from flow360.component.simulation.models.volume_models import (
+    ActuatorDisk,
+    BETDisk,
+    Fluid,
+    PorousMedium,
+    Rotation,
+    Solid,
+)
+from flow360.component.simulation.operating_condition.operating_condition import (
+    AerospaceCondition,
+    ThermalState,
+)
+from flow360.component.simulation.outputs.outputs import (
+    AeroAcousticOutput,
+    IsosurfaceOutput,
+    ProbeOutput,
+    SliceOutput,
+    SurfaceIntegralOutput,
+    SurfaceOutput,
+    SurfaceProbeOutput,
+    TimeAverageSurfaceOutput,
+    TimeAverageVolumeOutput,
+    VolumeOutput,
+)
+from flow360.component.simulation.primitives import Edge, ReferenceGeometry, Surface
+from flow360.component.simulation.simulation_params import SimulationParams
+from flow360.component.simulation.time_stepping.time_stepping import (
+    AdaptiveCFL,
+    RampCFL,
+    Steady,
+    Unsteady,
+)
+from flow360.component.simulation.unit_system import SI_unit_system
+from flow360.component.simulation.utils import _model_attribute_unlock
```

Comment:
This is internal used only. We should not expose this.

Reply:
removed.